src/components/AccountModule/WalletConnectionInfo.js

```diff
@@ -34,12 +43,25 @@ function ConnectionInfo() {
 
   const copyAddress = useCopyToClipboard(account, 'Address copied')
 
-  const Icon = hasNetworkMismatch ? IconCross : IconCheck
+  const { connectionMessage, connectionColor } = useWalletConnectionDetails(
+    clientListening,
+    walletListening,
+    clientOnline,
+    walletOnline,
+    clientSyncDelay,
+    walletSyncDelay,
+    walletNetworkName
+  )
+
+  const Icon = connectionColor !== theme.positive ? IconCross : IconCheck
 
+  const formattedConnectionMessage = connectionMessage.includes('Connected')
+    ? `Connected to Ethereum ${walletNetworkName} Network`
+    : connectionMessage
   return (
     <section
       css={`
-        max-width: ${42 * GU}px;
+        max-width: ${51 * GU}px;
```

Comment:
For the multiple providers,[ the width was changed to 51 GU to avoid the popover resizing over the connection step changes.](https://www.figma.com/file/bZ1j7omfhdMCbtFahP3nEJ9B/Client?node-id=1194%3A4010)

Reply:
Ahh gotcha! I'm just wondering if it makes to change our popovers to that width, since the amount of space to the right does look a bit awkward: 

<img width="432" alt="Screen Shot 2020-01-22 at 4 04 44 PM" src="https://user-images.githubusercontent.com/4166642/72905501-ed77d700-3d30-11ea-83fe-a247ff593009.png">

Especially since the select web3 popover will be a separate popover unrelated to our popovers for connection information.

Reply:
Hmmm maybe, but, there's now logic for handling mobile views, as the width changes to 328px I think. Maybe we could have a chat with @dizzypaty about this. :)

Reply:
Something else related to the Popover width: we should add a reasonable `min-width`, to prevent it to break visually on viewports that are narrower than 360px.

This is because `Popover` gets rendered on top of everything else and the `<Main />` min-width doesn’t apply to them.

Reply:
This could be something like `8 * 40 GU`? Even though that is the exact iPhone 5 / SE width.